torch/csrc/jit/interpreter.cpp

```diff
@@ -1053,6 +1053,14 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
       if (i > 0) {
         out << "during call ";
       }
+      int idx = 1;
```

Comment:
Can you format the error trace to more closely match python format. e.g. call it a Traceback have the string "(most recent call last)".  
I think it should also mention the function name being called in addition to the source range.